[2025-06-11T06:04:48.265+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: multi_region_ods_merge_v2.ods_merge_group.merge_customers scheduled__2025-06-10T00:00:00+00:00 [queued]>
[2025-06-11T06:04:48.279+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: multi_region_ods_merge_v2.ods_merge_group.merge_customers scheduled__2025-06-10T00:00:00+00:00 [queued]>
[2025-06-11T06:04:48.279+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-06-11T06:04:48.367+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): ods_merge_group.merge_customers> on 2025-06-10 00:00:00+00:00
[2025-06-11T06:04:48.376+0000] {standard_task_runner.py:60} INFO - Started process 2593 to run task
[2025-06-11T06:04:48.381+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'multi_region_ods_merge_v2', 'ods_merge_group.merge_customers', 'scheduled__2025-06-10T00:00:00+00:00', '--job-id', '1359', '--raw', '--subdir', 'DAGS_FOLDER/master_ods_dag.py', '--cfg-path', '/tmp/tmpe5p02uh9']
[2025-06-11T06:04:48.384+0000] {standard_task_runner.py:88} INFO - Job 1359: Subtask ods_merge_group.merge_customers
[2025-06-11T06:04:48.862+0000] {task_command.py:423} INFO - Running <TaskInstance: multi_region_ods_merge_v2.ods_merge_group.merge_customers scheduled__2025-06-10T00:00:00+00:00 [running]> on host 590b48c4a67f
[2025-06-11T06:04:49.090+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='multi_region_ods_merge_v2' AIRFLOW_CTX_TASK_ID='ods_merge_group.merge_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-06-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-10T00:00:00+00:00'
[2025-06-11T06:04:49.091+0000] {connection.py:486} INFO - Snowflake Connector for Python Version: 3.15.0, Python Version: 3.10.13, Platform: Linux-6.11.0-26-generic-x86_64-with-glibc2.36
[2025-06-11T06:04:49.092+0000] {connection.py:1391} INFO - Connecting to GLOBAL Snowflake domain
[2025-06-11T06:04:52.261+0000] {logging_mixin.py:188} INFO - Successfully connected to Snowflake via snowflake.connector.
[2025-06-11T06:04:52.261+0000] {logging_mixin.py:188} INFO - Running merge for customers...
[2025-06-11T06:04:53.944+0000] {logging_mixin.py:188} INFO - Merge completed for customers
[2025-06-11T06:04:54.523+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-11T06:04:54.540+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=multi_region_ods_merge_v2, task_id=ods_merge_group.merge_customers, execution_date=20250610T000000, start_date=20250611T060448, end_date=20250611T060454
[2025-06-11T06:04:54.574+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-11T06:04:54.620+0000] {taskinstance.py:3280} INFO - 2 downstream tasks scheduled from follow-on schedule check
